firstapp/Homework4.java

```diff
@@ -0,0 +1,123 @@
+package firstapp;
+
+public class Homework4 {
+    public static void main(String[] args) {
+        System.out.println("Task 1 CheckOddEven");
+        int fstnum = 12;
```

Comment:
vars name should be in camel case format.
For example firstNumber

Also, check below